tinygrad/tensor.py

```diff
@@ -262,7 +262,7 @@ def _data(self) -> memoryview:
     if self.device != "CLANG": buf.options = BufferSpec(nolru=True)
     return buf.as_buffer(allow_zero_copy=True if self.device != "CLANG" else False)
 
-  def data(self) -> memoryview:
+  def data(self) -> memoryview[Any]:
```

Comment:
Using `memoryview[Any]` instead of the proper narrow `Union[memoryview[float], memoryview[int], memoryview[bool]`. This saves a bunch of lines without much cheating imo.

Reply:
Why is it more lines? You can use `|` for union